internal/factsengine/gatherers/saptune.go

```diff
@@ -70,22 +70,21 @@ func (s *SaptuneGatherer) Gather(factsRequests []entities.FactRequest) ([]entiti
 	facts := []entities.Fact{}
 	log.Infof("Starting %s facts gathering process", SaptuneGathererName)
 	saptuneRetriever, err := saptune.NewSaptune(s.executor)
-
+	if err != nil {
+		return facts, &SaptuneNotInstalled
```

Comment:
We could return `nil, &SaptuneNotInstalled` as well (the same for the next)
And even `SaptuneNotInstalled.Wrap(err.Error())` in this case, as we have the rpm command error